src/components/VmDetail/index.js

```diff
@@ -73,7 +75,14 @@ class VmDetail extends Component {
 VmDetail.propTypes = {
   vm: PropTypes.object,
   icons: PropTypes.object.isRequired,
-  actions: PropTypes.object.isRequired,
+  onCloseVmDetail: PropTypes.func.isRequired,
 }
 
-export default VmDetail
+export default connect(
+  (state) => ({
+    icons: state.icons,
+  }),
+  (dispatch, ownProps) => ({
```

Comment:
no need to take the `ownProps` argument if it is not used a plain `(dispatch) => ({ ...` is sufficient